decidim_hospitalet-surveys/config/locales/ca.yml

```diff
@@ -1,5 +1,14 @@
 ---
 ca:
+  activerecord:
+    errors:
+      models:
+        decidim_hospitalet/surveys/survey_result:
+          attributes:
+            feature:
+              required: es obligatori
```

Comment:
I was in the middle of a review but this PR was merged :/ 

`s/es/és/` in both occurrences